smoke-tests/apps/TelemetryProcessors/src/smokeTest/resources/applicationinsights-mask-log-body.json

```
@@ -0,0 +1,55 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "processors": [
+      {
+        "type": "log",
+        "body": {
+          "toAttributes": {
+            "rules": [
+              "userId (?<redactedUserId>[\\da-zA-Z]+)"
+            ]
+          }
+        },
+        "id": "/log/extractRedactedUserId"
+      },
+      {
+        "type": "attribute",
+        "actions": [
+          {
+            "key": "redactedUserId",
+            "action": "delete"
+          }
+        ],
+        "id": "/attribute/deleteRedactedUserId"
+      },
+      {
+        "type": "log",
+        "body": {
+          "toAttributes": {
+            "rules": [
+              "This is my \"email\" : \"(?<redactedEmail>.*)\""
```

this is probably more typical (and safer)
```suggestion
              "This is my \"email\" : \"(?<redactedEmail>[^\"]*)\""
```